commonwords.py

```diff
@@ -0,0 +1,86 @@
+import wikipedia
+import math
+
+def text_to_word(text):
+    """
+    Purpose: To refine the word being available to do further process.
+    The function replaces non-alphabet characters into space or blank as well as change all into lower character.
+    The string typed text is divided into multiple words by using split function.
+    """
+    text = text.replace('-',' ')
+    text = text.replace('(','')
+    text = text.replace(')','')
+    text = text.replace("'",'')
+    text = text.replace(".",'')
+    text = text.replace(",",'')
+    text = text.lower()
+    list_of_unsorted = text.split()
+    return list_of_unsorted
+
+def histogram(unsorted_words):
+    """
+    produces a dictionary composed of common word from two summary
+    in which the key and value is the word and the number of word.
+    """
+    count = dict()
+    for word in unsorted_words:
+        if word not in count:
+            count[word] = 1
+        else:
+            count[word] += 1
+    return count
```

Comment:
(Also minor suggestion)
In python library, there is a special dictionary called `Counter`.
By default, this initializes the count to be 0, so that you wouldn't need to check if the keyword is inside the dictionary or not.
An equivalent code using Counter will be like following
```
from collections import Counter
count = Counter()
for word in unsorted_words:
    count[word] += 1
return count
```